docs/src/main/sphinx/release/release-411.md

```diff
@@ -1,27 +1,53 @@
-# Release 411 (15 Mar 2023)
+# Release 411 (30 Mar 2023)
 
 ## General
 
 * Add spilled data size to query statistics. ({issue}`16442`)
 * Add {func}`sinh` function. ({issue}`16494`)
+* Add {func}`quantile_at_value` function. ({issue}`16736`)
 * Add support for a `GRACE PERIOD` clause in the `CREATE MATERIALIZED VIEW`
   task. This is a backwards incompatible change, and the previous behavior can
   be restored with the `legacy.materialized-view-grace-period` configuration
   property or the `legacy_materialized_view_grace_period` session property. ({issue}`15842`)
-* Add `CANCELING`, `ABORTING`, and `FAILING` task statuses for tasks which are
-  waiting on running drivers to fully terminate. ({issue}`15478`)
-* Improve error messages for exceptions thrown by a connector while querying
-  metadata. ({issue}`16497`)
-* Fix generation of IDs for `keyvalue()` methods in the
-  [JSON path](json-path-language). Previously, separate IDs were generated for
-  each member. ({issue}`16482`)
+* Fix potential incorrect query stats when tasks are waiting on running drivers
+  to fully terminate. ({issue}`15478`)
+* Add support for specifying the number of nodes that will write data during
+  `INSERT`, `CREATE TABLE AS SELECT`, or `EXECUTE` queries with the
+  `query.max-writer-tasks-count` configuration property. ({issue}`16238`)
+* Improve performance of queries that contain predicates involving the `year`
+  function. ({issue}`14078`)
+* Improve performance of queries that contain a `sum` aggregation. ({issue}`16624`)
+* Improve performance of `filter` function on arrays. ({issue}`16681`)
+* Reduce memory usage of the coordinator. ({issue}`16668`, {issue}`16669`)
+* Reduce redundant data exchanges for queries with multiple aggregations. ({issue}`16328`)
+* Fix incorrect query results when using `keyvalue()` methods in the
+  [JSON path](json-path-language). ({issue}`16482`)
+* Fix potential incorrect results in queries involving joins and a
+  non-deterministic value. ({issue}`16512`)
+* Fix potential query failure when exchange compression is enabled. ({issue}`16541`)
+* Fix query failure when calling a function with a large of parameters. ({issue}`15979`)
+
+## BigQuery connector
+
+* Fix failure of aggregation queries when executed against a materialized view,
+  external table, or snapshot table. ({issue}`15546`)
 
 ## Delta Lake connector
 
 * Add support for inserting into tables that have
-  [column invariants](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-invariants). ({issue}`16136`)
+  [simple invariants](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-invariants). ({issue}`16136`)
+* Add the generated expression to the `Extra` column in the results of
```

Comment:
What's a "generated expression"?

Reply:
I believe it's [generated column expressions](https://docs.delta.io/latest/delta-batch.html#use-generated-columns)?
cc @ebyhr